pkg/config/templates/cos-rootfs.yaml

```diff
@@ -5,7 +5,7 @@ environment_file: /run/cos/cos-layout.env
 environment:
   VOLUMES: "LABEL=COS_OEM:/oem LABEL=COS_PERSISTENT:/usr/local{{ if .ShouldMountDataPartition }} LABEL=HARV_LH_DEFAULT:/var/lib/harvester/defaultdisk{{ end }}"
   OVERLAY: "tmpfs:25%"
-  RW_PATHS: "/var /etc /srv /boot"
+  RW_PATHS: "/var /etc /srv /boot{{ if .OS.RWPaths }}{{- range $path := .OS.RWPaths }} {{ $path }}{{- end }}{{ end }}"
```

Comment:
Would this also take effect with day2 configurations?

Reply:
can change it in /oem/90_custom.yaml

Reply:
Without this PR, users can achieve the same effect by modifying `/oem/90_custom.yaml` and then rebootting. This PR allows users to define the desired os layout at installation time.

Reply:
removed the rwpath option